index.html

```diff
@@ -85,6 +85,7 @@ <h2>Who?</h2>
             <li><a title="Codecov" href="https://github.com/codecov/self-hosted" ><img src="./logos/codecov.svg" alt="Codecov logo" /></a></li>
             <li><a title="Convex" href="https://github.com/get-convex/convex-backend" ><img src="./logos/convex.svg" alt="Convex logo" /></a></li>
             <li><a title="GitButler" href="https://github.com/gitbutlerapp/gitbutler" ><img src="./logos/gitbutler.svg" alt="GitButler logo" /></a></li>
+            <li><a title="PowerSync" href="https://github.com/powersync-ja/" ><img src="./logos/powersync.svg" alt="PowerSync logo" /></a></li>
```

Comment:
Can we use https://github.com/powersync-ja/powersync-service instead of the org link? Afaict from reading the [blog post](https://www.powersync.com/blog/new-open-era-for-powersync) that repo is the one licensed under FSL, yes?

Reply:
Hi @chadwhitacre thanks for the swift response. I have made the change :)